Python/symtable.c

```diff
@@ -2279,6 +2286,25 @@ symtable_visit_expr(struct symtable *st, expr_ty e)
         if (!symtable_raise_if_annotation_block(st, "await expression", e)) {
             VISIT_QUIT(st, 0);
         }
+        if (!allows_top_level_await(st)) {
+            if (!_PyST_IsFunctionLike(st->st_cur)) {
+                PyErr_SetString(PyExc_SyntaxError,
+                                "'await' outside function");
```

Comment:
If you use a macro for ASYNC_FOR_OUTSIDE_ASYNC_FUNC, maybe also use a macro for this error? (though I don't know if the macro is used more than once actually).

Reply:
Yes, the error message is only used once, so I don't see much point in adding a macro. I suppose we could get rid of the other single-use macros too but I try to avoid unnecessary changes.

Reply:
Would you be interested in a PR removing them?

Reply:
I don't think so, we don't generally accept purely cosmetic PRs.